Module.php

```diff
@@ -69,6 +67,14 @@ public function showAgeCheck()
         return (boolean)$this->settings->get('showAgeCheck', false);
     }
 
+    /**
+     * @return string
+     */
+    public function defaultAge()
+    {
+        return $this->settings->get('defaultAge', '');
```

Comment:
Why not `getDefaultAge()` and why not return a default value in the second argument?

Reply:
I went for simple and short `defaultAge`, as for returning a value, doing so would defeat the purpose of allowing a custom value wouldn't it? :thinking:

Reply:
Changed